### ISSUE TITLE
Question description only shown in advanced panel but not in modal dialog

### ISSUE THREAD

**Issue by glumb** [surveyjs/survey-creator#602]
### Are you requesting a feature, reporting a bug or ask a question?
feature

### What is the current behavior?
question description can not be set from modal edit dialog.

### What is the expected behavior?
textfield for setting the description.

![Bildschirmfoto 2019-09-21 um 13 02 53](https://user-images.githubusercontent.com/3062564/65372422-5ca0ca00-dc70-11e9-8fc8-f030eff81f05.png)

**Comment by andrewtelnov** [comment]
@glumb I am afraid having description would not help Property Grid a lot. We think to rewrite this Property Grid completely. Based on our experience during UI usability testing and based on our stats,  end-users rarely use Property Grid and use only Modal Window or try to make the changes directly in designer area. In fact, PropertyGrid concepts is known more by software developers, others use to work with other type of UI.

We believe, that probably by adding categories, and making it looks like more like a Modal Editor may change the situation. 
Several companies shows their design of Property Grid. How they want it to look. Unfortunately, their concepts works great for simple questions (editor/dropdown). These companies wanted to avoid Modal Windows completely.

The bottom line, Property Grid needs help, but description would not change anything without categories and other changes.

**Comment by glumb** [comment]
We want to hide the property grid for our users. But the question description can only be set from the property grid. 
I was suggesting to also add the description property to the modal dialogue.

**Comment by andrewtelnov** [comment]
@glumb Right, I thought too much about Property Grid and did not realize that you are talking about question description and not properyt description.
I have added question description in the commit above. It will be there from the next minor version that we will release on the next week.
You may customize Question Editor properties as you need. Here is the [doc](https://surveyjs.io/Documentation/Survey-Creator/#elementseditor)

Thank you,
Andrew

**Comment by glumb** [comment]
Perfect! thanks also for the prompt reply :)
